### ISSUE TITLE
 'URLSearchParams' is not defined 

### ISSUE THREAD

**Issue by kongakong** [eslint/eslint#5668]
**What version of ESLint are you using?** 

2.4

**What parser (default, Babel-ESLint, etc.) are you using?** 

babel-eslint ^6.0.0-beta.6

**Please show your full configuration:** 

eslint-config-airbnb 6.1.0

**What did you do? Please include the actual source code causing the issue.**

`const searchParams = new URLSearchParams();`

**What did you expect to happen?**

`URLSearchParams ` should not be flagged as undefined because it is part of WebAPIs

**What actually happened? Please include the actual, raw output from ESLint.**

 13:30  error  'URLSearchParams' is not defined  no-undef

**Comment by ilyavolodin** [comment]
https://github.com/sindresorhus/globals/pull/83
Once next version of globals will be release this issue will be fixed automatically.

**Comment by mysticatea** [comment]
We have `globals@^8.12` in our package.json, and the latest `globals` is v9.2.0.
So we have to upgrade the dependency.

**Comment by tysonnero** [comment]
I'm seeing this same issue using the following versions:
eslint: 3.4.0
babel-eslint: 6.1.2

Just wondering if I'm possibly doing something wrong.

**Comment by esindger** [comment]
I still have this error.
```
└─┬ eslint@3.19.0
  └── globals@9.17.0 
```

**Comment by ilyavolodin** [comment]
@tysonnero Try reinstalling eslint. It's possible you are pulling older version of globals
